packages/components/organisms/f-checkout/test/specs/accessibility/axe-accessibility.spec.js

```diff
@@ -1,15 +1,43 @@
 import { getAccessibilityTestResults } from '../../../../../../../test/utils/axe-helper';
-import CheckoutComponent from '../../../test-utils/component-objects/f-checkout.component';
+const Checkout = require('../../../test-utils/component-objects/f-checkout.component');
+const checkout = new Checkout();
```

Comment:
Could we tweak the naming here, something like this maybe?

```suggestion
const CheckoutComponent = require('../../../test-utils/component-objects/f-checkout.component');
const checkoutPage = new CheckoutComponent();
```

Reply:
Ah I see this is what it used to be, what was the reason for changing it?

Reply:
ah so I changed it so it matched the class name - some of the functions mention the component so I thought it would run smoother writing it as checkout.isComponentDisplayed. What d'you think?

Reply:
Discussed on Slack, we'll leave this as-is